packages/interface/.env.example

```diff
@@ -26,20 +26,12 @@ NEXT_PUBLIC_WALLETCONNECT_ID="21fef48091f12692cad574a6f7753643"
 # Event title for the round, just for display
 NEXT_PUBLIC_EVENT_NAME="Devcon"
 
-# Unique identifier for your applications and lists - your app will group attestations by this id
-NEXT_PUBLIC_ROUND_ID="open-rpgf-1"
-# Event title for the round, just for display
-NEXT_PUBLIC_ROUND_ORGANIZER="PSE"
+# Event description, just for display
+NEXT_PUBLIC_EVENT_DESCRIPTION="Write a descripion about your community"
 
 # Name of the token you want to allocate (only updates UI)
 NEXT_PUBLIC_TOKEN_NAME="Votes"
 
-# Voting periods
-# Determine when users can register applications, admins review them, voters vote, and results are published
-NEXT_PUBLIC_START_DATE=2024-01-01T00:00:00.000Z
-NEXT_PUBLIC_REGISTRATION_END_DATE=2024-01-01T00:00:00.000Z
-NEXT_PUBLIC_RESULTS_DATE=2024-01-01T00:00:00.000Z
```

Comment:
Let's remove these fields for workflows and after these changes are merged into main, let's remove them from repo settings for actions and dependabot